StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PaymentSheetVerticalViewController.swift

```diff
@@ -233,19 +240,49 @@ class PaymentSheetVerticalViewController: UIViewController, FlowControllerViewCo
         configuration.style.configure(self)
 
         // One stack view contains all our subviews
-        let views: [UIView] = [headerView, walletHeaderView, paymentContainerView].compactMap { $0 }
+        let views: [UIView] = [headerLabel, walletHeaderView, paymentContainerView].compactMap { $0 }
         let stackView = UIStackView(arrangedSubviews: views)
         stackView.directionalLayoutMargins = PaymentSheetUI.defaultMargins
         stackView.isLayoutMarginsRelativeArrangement = true
         stackView.axis = .vertical
-        stackView.setCustomSpacing(24, after: headerView)
+        stackView.setCustomSpacing(24, after: headerLabel)
         if let walletHeaderView {
             stackView.setCustomSpacing(24, after: walletHeaderView)
         }
 
         view.addAndPinSubview(stackView, insets: .init(top: 0, leading: 0, bottom: PaymentSheetUI.defaultSheetMargins.bottom, trailing: 0))
     }
 
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        walletPaymentOption = nil
+    }
+
+    // MARK: - PaymentSheetViewControllerProtocol
+
+    func pay(with paymentOption: PaymentOption, animateBuyButton: Bool) {
+        paymentSheetDelegate?.paymentSheetViewControllerShouldConfirm(self, with: paymentOption) { [weak self] result, _ in
```

Comment:
I'm implementing this as well FYI with the buy button

Reply:
ACK, I don't plan to implement much more than what is stubbed here.